eng/pipelines/dailytests.yml

```
@@ -0,0 +1,31 @@
+resources:
+  pipelines:
+  - pipeline: ComponentBuildUnderTest
+    source: NuGet.Client-Official
+    trigger:
+        branches:
+          - dev-*
+
+parameters:
+- name: DartLabEnvironment
+  displayName: DartLab Environment
+  type: string
+  default: Production
+  values:
+  - Production
+  - Staging
+- name: ApexAgentCleanup
+  displayName: Delete or keep VS Apex test machine for debugging
+  type: string
+  default: delete
+  values:
+  - delete
+  - stop
+
+extends:
+  template: templates/dailytests.yml
+  parameters:
+    DartLabEnvironment: ${{parameters.DartLabEnvironment}}
+    ApexAgentCleanup: ${{parameters.ApexAgentCleanup}}
+    NuGetLocalizationType: Full
+    isOfficialBuild: true
```

```suggestion
    isOfficialBuild: false
```

Official builds are those that create binaries that we ship to customers. Test pipelines don't create any artifacts that we ship to customers, so are never "official" builds.

For our official builds, we use the variable/parameter to prevent flaky tests breaking the build, and therefore making it impossible to create a release pipeline from the build. But since this test pipeline won't be the input of another pipeline, the same requirement doesn't exist.